src/main/java/it/gov/pagopa/pu/classification/service/AssessmentsService.java

```diff
@@ -14,14 +20,41 @@
 public class AssessmentsService {
 
   private final InstallmentNoPIIService installmentNoPIIService;
+  private final IngestionFlowFileService ingestionFlowFileService;
+  private final DebtPositionTypeOrgService debtPositionTypeOrgService;
+  private final AssessmentsRepository assessmentsRepository;
 
-
-  public AssessmentsService(InstallmentNoPIIService installmentNoPIIService) {
+  public AssessmentsService(InstallmentNoPIIService installmentNoPIIService, IngestionFlowFileService ingestionFlowFileService, DebtPositionTypeOrgService debtPositionTypeOrgService, AssessmentsRepository assessmentsRepository) {
     this.installmentNoPIIService = installmentNoPIIService;
+    this.ingestionFlowFileService = ingestionFlowFileService;
+    this.debtPositionTypeOrgService = debtPositionTypeOrgService;
+    this.assessmentsRepository = assessmentsRepository;
   }
 
   public List<InstallmentNoPIIResponse> getInstallmentsByReceiptId(Long receiptId, String accessToken) {
     return installmentNoPIIService.getByReceiptId(receiptId, accessToken);
   }
 
+  public Assessments getAssessment(InstallmentNoPIIResponse installmentNoPIIResponse, String accessToken) {
```

Comment:
why this name? this is not a get, this is a build.
Moreover, I don't see a real reason to have this method public... this is should be a private method...
(move it after it's usage, in order to have a right read order)